src/components/home/UserInput.vue

```diff
@@ -182,7 +182,7 @@ export default Vue.extend({
       //just a few entries in geneList to show how the parsing works
       this.clear().then(() => {
         this.demoRunning = true
-        const text = 'TP53\nBRCA1,BRCA2\nNOT_A_GENE, ALK_FUSION\nBRAF1'.split(
+        const text = 'TP53\nBRCA1,BRCA2\nABCDE, ALK_FUSION\nBRAF1, BCL6'.split(
```

Comment:
Let's also use a common entry in a bed files for a gene location that's not a gene, along with ABCDE. Let's use something like RS7525559 (on the same line) which is in many bed files (because it's a legit location, just not a real gene name).
Let's put the fusion on it's own line

Reply:
done